apps/cookbook/src/app/iphone/iphone.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="container">
   <div class="docs-demo-device ios">
     <figure>
-      <svg class="docs-demo-device__md-bar" viewBox="0 0 1384.3 40.3">
+      <svg class="docs-demo-device-md-bar" viewBox="0 0 1384.3 40.3">
```

Comment:
Technically this seems to be ok - since the corresponding CSS class selector is also renamed. However, it may or may not be what we want. We haven't discussed naming conventions like BEM, have we?

Reply:
This is changed for consistency across the code base. If we opt for something like BEM, I think we should revisit.